libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java

```diff
@@ -124,6 +124,54 @@ static void rerouteEvent(SessionState sessionState, RouteProgress routeProgress,
     ));
   }
 
+  static void feedbackEvent(SessionState sessionState, RouteProgress routeProgress, Location location,
+                            String description, String feedbackType, String screenshot) {
+    String upcomingInstruction = null;
```

Comment:
The code from L129 to L156 is the same exact code implemented in `rerouteEvent` 👀 https://github.com/mapbox/mapbox-navigation-android/blob/e7ae10d673d61dc4a1e91d26ddb4de7ae3c1197a/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMetricsWrapper.java#L72-L99
What about avoiding duplicate code extracting that block into a shared method?